script.js

```diff
@@ -1,4 +1,211 @@
+// Global variables
+var arrayActive = [];
+var scoreBoard = [];
+var numberOfRolls = 2;
+var numberOfPlayers = 2;
+var lastPlayer = 0;
+var currentPlayer = 1;
+var scoreBoardLock = 0;
+var checkWinner = "";
+
+// Game states: settings, roll dice, choose order
+var gameState = "settings";
+
+// Game settings
+var settingsList =
+  "</br>Settings: autoplay, lowest, play X</br>To start, leave input blank";
+var autoPlay = 0;
+var lowest = 0;
+
+// Main function
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (gameState == "settings") {
+    if (input == "autoplay") {
+      autoPlay = 1;
+      return `Autoplay enabled!${settingsList}`;
+    }
+
+    if (input == "lowest") {
+      lowest = 1;
+      return `Lowest wins enabled!${settingsList}`;
+    }
+
+    if (input.split(" ")[0] == "play") {
+      numberOfPlayers = input.split(" ")[1];
+      return `There are ${numberOfPlayers} players.${settingsList}`;
+    }
+
+    if (!input == "") {
+      return `That's not a setting!${settingsList}`;
+    }
+
+    gameState = "roll";
+    return `Press submit to roll a dice for Player ${currentPlayer}.`;
+  }
+
+  if (gameState == "roll") {
+    arrayActive = generateDiceArray(numberOfRolls);
+    gameState = "order";
+    return `Player ${currentPlayer} rolled ${arrayActive[0]} and ${arrayActive[1]}.`;
+  }
```

Comment:
Good job simplifying this part of the game by using helper functions as well as being able to repeat this block of code for multiple players